main.go

```diff
@@ -75,7 +93,7 @@ func (s *service) start() error {
 	listener, err := net.Listen("tcp", s.conf.Listen)
 
 	if err != nil {
-		return err
+		log.Fatalln(err)
```

Comment:
Just return the error here, you already have `log.Fatal` in `main` function for this.